content/en/docs/collector/deployment/gateway/index.md

```diff
@@ -198,6 +199,30 @@ The load-balancing exporter emits metrics including
 that you can use for health and performance monitoring of the OTLP endpoint
 collector.
 
+## Typical Gateway deployment architecture
```

Comment:
What makes that deployment "typical"? I just want to verify that this is not a too strong wording, maybe "exemplary" or "Example of a gateway deployment architecture" or "Example architecture"?

Reply:
Added some description. Please take a look.

Reply:
I still struggle with the "typical" since it is very prescriptive while not giving the end user a hint what we are talking about. Reading your updated text (thanks for that!) I rather prefer something like

```suggestion
## Combined deployment of Collectors in gateway and agent mode
```

It's rather long so maybe there is a way to express this shorter, but the intend is clearer that this is a section that talks about combining the modes